### ISSUE TITLE
Can't store resized pdf as jpg

### ISSUE THREAD

**Issue by dedekm** [markevans/dragonfly#483]
```
Command failed ('convert' '/tmp/dragonfly20180309-24076-u498ms' '-quality' '90' '/tmp/dragonfly20180309-24076-zurgvd.jpg') with exit status 1 and stderr convert: no decode delegate for this image format `' @ error/constitute.c/ReadImage/501.
convert: no images defined `/tmp/dragonfly20180309-24076-zurgvd.jpg' @ error/convert.c/ConvertImageCommand/3210.
```
I'm trying to resize first page of PDF to JPG and then store it at s3.
`pdf_file.thumb('200x200', format: :jpg, frame: 0)` works fine and produce resized thumbnail of pdf.
but `pdf_file.thumb('200x200', format: :jpg, frame: 0).store` produce invalid file!

**Comment by dedekm** [comment]
i found the solution
`pdf_file.thumb('200x200', 'format' => :jpg, 'frame' => 0).store` works because imagemagick plugin accepts only string keys

this is the file where is cause:
[lib/dragonfly/image_magick/processors/convert.rb](https://github.com/markevans/dragonfly/blob/master/lib/dragonfly/image_magick/processors/convert.rb)

**Comment by stahor** [comment]
+ 1 
I wasted 2 hours trying to understand what happens. This have to be described in documentation. Or it's even better to replace options hash with hash with indifferent access

**Comment by markevans** [comment]
hash with indifferent access is a rails thing.
I specifically don't want to convert between symbols and strings because it adds complexity in other areas.
String keys were chosen because they easily JSON serialize/deserialize.
When you say it should be described in the documentation, I'm not sure what you mean as in http://markevans.github.io/dragonfly/imagemagick it clearly uses string keys.
Do you mean there should be a note of emphasis? If so where would you put it? The problem is of course that I have to balance catering for people overlooking things with keeping the docs clear and concise, which is no easy feat! ;)

**Comment by dedekm** [comment]
you are right that in the documentation iit uses string keys, didn't noticed that...
but `image.thumb` works with symbol keys, `.store` doesn't, so thats why I was confused

**Comment by stahor** [comment]
What about adding your own implementation of `stringify_keys` for example?
